lib/fathead/mdn_css/output.txt

```diff
@@ -375,23 +543,52 @@
 @supports	A										<section class="prog__container"><p>The @supports CSS at-rule associates a set of nested statements, in a CSS block, that is delimited by curly braces, with a condition consisting of testing of CSS declarations, that is property-value pairs, combined with arbitrary conjunctions, disjunctions, and negations of them. Such a condition is called a supports condition.</p><pre><code>( transform-origin: 5% 5% )</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@supports
+@swash	A										<section class="prog__container"><p>Specifies a feature name that will work with the swash() functional notation of font-variant-alternates. A swash feature value definition allows only one value: ident1: 2 is valid when ident2: 2 4 isn't.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@font-feature-values#@swash
 @viewport	A										<section class="prog__container"><p>The @viewport CSS at-rule contains a set of nested descriptors in a CSS block that is delimited by curly braces. These descriptors control viewport settings, primarily on mobile devices.</p><pre><code>@viewport {\n <group-rule-body>\n}</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@viewport
-Hz	A										<section class="prog__container"><p>Hz represents a frequency in Hertz. E.g. 0Hz, 1500Hz, 10000Hz.</p><pre><code>12Hz Positive integer.\n-456kHz Negative integer.\n4.3Hz Non-integer.\n14KhZ The unit is case-insensitive, though non-SI capitalization is not recommended.\n+0Hz Zero, with a leading + and the unit.\n-0kHz Zero, with a leading - and the unit (Though strange, this is an allowed value).\n</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/frequency#Hz
-active	R	:active										
+CSSRule	R	cssrulelist										
+CSSRule.selectorText	R	cssstylerule.selectortext										
+CSSRuleList	R	cssrulelist										
+CSSStyleSheet	R	cssrule.csstext										
+U+2026 Horizontal Ellipsis	R	text-overflow										
+[ <line-names>? <string> <track-size>? <line-names>? ]+ [ / <explicit-track-list> ]?	A										<section class="prog__container"><p>Sets grid-template-areas to the strings listed, grid-template-rows to the track sizes following each string (filling in auto for any missing sizes), and splicing in the named lines defined before/after each size, and grid-template-columns to the track listing specified after the slash (or none, if not specified).\n  \n Note: The repeat() function isn’t allowed in these track listings, as the tracks are intended to visually line up one-to-one with the rows/columns in the “ASCII art”.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template#Values
```

Comment:
@hchienjo I'm not sure we should be capturing this information -- these are very unlikely to ever surface